meson.build

```
@@ -1,7 +1,7 @@
 project(
   'matplotlib',
   'c', 'cpp',
-  version: '3.9.0.dev0',
+  version: run_command(find_program('python3'), '-m', 'setuptools_scm').stdout().strip(),
```

```suggestion
  version: run_command(find_program('python3'), '-m', 'setuptools_scm', check: true).stdout().strip(),
```
will get rid of a warning.

---

Sorry for spamming, but this may actually be the key thing. If `setuptools_scm` is not available, not having `check: true` will just continue running the install script. My current guess is that `setuptools_scm` is not installed in the failing work flows.

(I *think* I had an issue with tags as well, but this was the final boss in my case.)